ci/numba_cuda.diff

```diff
@@ -0,0 +1,14 @@
+diff --git a/numba_cuda/numba/cuda/cudadrv/driver.py b/numba_cuda/numba/cuda/cudadrv/driver_new.py
+index 1641bf7..233e9ed 100644
+--- a/numba_cuda/numba/cuda/cudadrv/driver.py
++++ b/numba_cuda/numba/cuda/cudadrv/driver.py
+@@ -365,6 +365,9 @@ class Driver(object):
+         else:
+             variants = ('_v2', '')
+
++        if fname in ("cuCtxGetDevice", "cuCtxSynchronize"):
++            return getattr(self.lib, fname)
++
+         for variant in variants:
+             try:
+                 return getattr(self.lib, f'{fname}{variant}')
```

Comment:
Just for reference, the fix comes from: https://github.com/NVIDIA/numba-cuda/pull/185/files (as per Graham, this being the likely fix -- and it was :)).